src/classes/GE_PaymentServices.cls

```diff
@@ -100,8 +100,8 @@ public with sharing class GE_PaymentServices {
      * @description Returns Elevate Product Id to payments services tokenize card iframe
      * @return String
      */
-    public String getProductId() {
-        return configService.getProductId();
+    public String getClientId() {
```

Comment:
@sokosaurus This update was originally part of Victor's work item and I *believe* should also be reflected in the parent branch of this PR. I think this is maybe only showing up as a diff here as a weird glitch in github - 

https://github.com/SalesforceFoundation/NPSP/blob/feature/232/src/classes/GE_PaymentServices.cls#L103

I'm pointing that out only because it's throwing me to see this in the diff, but I'm going to ignore it as something goofy github is doing unless you have any further insight!